lib/Resource.js

```diff
@@ -245,6 +245,9 @@ class Resource {
 
 	/**
 	 * Gets the resources stat info.
+	 * Note that a resources stat information is not updated when the resource is being modified.
+	 * Also, depending on the used adapter, some fields might be missing which would be present for a
+	 * [fs.Stats]{@link https://nodejs.org/api/fs.html#fs_class_fs_stats} instance.
```

Comment:
Maybe refer to `#getSize`? This PR doesn't seem to be up to date, so this method is missing here.

Reply:
getSize is currently not public. Should we change that? 🤔

Reply:
As discussed: Let's keep it private for the time being and until we used it for a while in our own code.